src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -92,6 +116,123 @@ export default class DOWLandingPage extends Vue {
     }
   ];
 
+  get doesCurrentEnvExist(): boolean {
+    return CurrentEnvironment.currentEnvironment && 
+            CurrentEnvironment.currentEnvironment.current_environment_exists === "YES";
+  }
+
+  /**
+   * Sets the 'label' and 'isComplete' properties of the optimize/replicate section.
+   *
+   * Label gets built using the classification level options that the user selects. Default label
+   * is left as is, if no levels are selected
+   *
+   * Current functions are complete, if all the below are true;
+   * 1. If the user chooses NOT to optimize or replicate
+   * 2. If the user chooses to optimize or replicate AND
+   *      chooses (NO) or (YES for additional growth and enters anticipated capacity)
+   * 3. If the user chooses to optimize or replicate AND
+   *      chooses (NO) or (YES for phased approach and enters approach schedule)
+   */
+  setCurrentFunctionsProperties(): void {
+    const currEnv = CurrentEnvironment.currentEnvironment;
+    // is complete check
+    const reqSectionIndex =this.requirementSections
+      .findIndex(reqSection => reqSection.section === "ReplicateOptimize")
+    this.requirementSections[reqSectionIndex].isComplete =
+        (currEnv.current_environment_replicated_optimized === "NO") ||
+        ((currEnv.statement_replicated_optimized !== "") &&
+            ((currEnv.additional_growth === 'NO') || (currEnv.additional_growth === 'YES' &&
+                (currEnv.anticipated_yearly_additional_capacity as unknown as string) !== "")) &&
```

Comment:
when `anticipated_yearly_additional_capacity` has no value it is `null` not empty string

Reply:
It is null initially, but when the value is entered and then is cleared, it is getting set to empty string.  Resolved it by adding both the conditions.